src/utils.tsx

```diff
@@ -369,3 +369,27 @@ export function isEqualUnsorted<T>(array: T[], includes: T[]): boolean {
     if (array.length !== includes.length) return false
     return includes.every((i) => array.includes(i))
 }
+
+export const getWeatherDescriptionFromApi = async (
+    iconName: string,
+): Promise<string> => {
+    const weatherNameMatch = iconName.match(/.+?(?=_|$)/)
+    const url = `https://api.met.no/weatherapi/weathericon/2.0/legends`
+    return await fetch(url)
```

Comment:
Her er det blanding av async/await og .then/.catch. Det er nok med éin av delane, og eg vil anbefale async await. Då må du også bruke try/catch.

Reply:
Visste ikke om at kun en trenger å brukes, takk for info!:) Har endret den til å kun bruke async/await med try/catch nå